waku/waku_archive/driver/sqlite_driver/queries.nim

```diff
@@ -117,39 +105,15 @@ proc createOldestMessageTimestampIndex*(db: SqliteDatabase): DatabaseResult[void
     )
   return ok()
 
-proc createHistoryQueryIndexQuery(table: string): SqlQueryStr =
-  "CREATE INDEX IF NOT EXISTS i_query ON " & table &
-    " (contentTopic, pubsubTopic, storedAt, id);"
```

Comment:
Why do we remove this index? it doesn't add value anymore?

Reply:
We used this in v2 but not anymore. `storedAt` and `id` don't exist anymore either.

Reply:
We will probably need new indexes. I'm open to suggestions.

Reply:
Makes sense! Yes, most likely searching by content or pubsub topic will happen pretty often. I would focus on the indexes separately as future optimizations, though

Reply:
> Makes sense! Yes, most likely searching by content or pubsub topic will happen pretty often. I would focus on the indexes separately as future optimizations, though

Yes, performance review of the Postgres driver/storev3 is planned. I think having perf. data will help a lot in writing good indexes.